sormas-backend/src/main/java/de/symeda/sormas/backend/selfreport/SelfReportFacadeEjb.java

```diff
@@ -77,6 +78,8 @@ public class SelfReportFacadeEjb
 
 	@EJB
 	private LocationFacadeEjbLocal locationFacade;
+	@EJB
+	private SelfReportService selfReportService;
```

Comment:
SelfReportService is already injected, so this EJB is not needed

Reply:
removed